app/controllers/api/v1/movies_controller.rb

```diff
@@ -7,4 +7,23 @@ def index
   def show
     render json: Movie.find(params[:id]), serializer: MovieShowSerializer
   end
+
+  def create
+    movie = Movie.new(movie_params)
+    if movie.save
+      saved = {movieSaved: true, response: movie.id}
+      render json: saved
```

Comment:
if the save was successful we should just send back the new Movie that was created (which follows standard HTTP) - React can look at the info sent back and look if it was a movie or errors